lambda/client.py

```diff
@@ -0,0 +1,53 @@
+import requests
+
+
+class Client:
+    def __init__(self, url: str, api_key: str, model: str):
+        self.url = url
+        self.api_key = api_key
+        self.model = model
+
+    def api_request(self, prompt: str, question: str) -> dict:
+        payload = {
+            "model": self.model,
+            "messages": [
+                {
+                    "role": "system",
+                    "content": [{"type": "text", "text": prompt}],
+                },
+                {
+                    "role": "user",
+                    "content": [{"type": "text", "text": question}],
+                },
+            ],
+        }
+
+        response = requests.post(
+            url=self.url,
+            headers=self._headers(),
```

Comment:
```suggestion
            headers=self._api_headers(),
```

To respect the change suggestion below.